hack/vendor.sh

```diff
@@ -29,7 +29,7 @@ clone git github.com/RackSec/srslog 6eb773f331e46fbba8eecb8e794e635e75fc04de
 clone git github.com/imdario/mergo 0.2.1
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork v0.6.1-rc2
+clone git github.com/docker/libnetwork v0.7.0-dev.2
```

Comment:
to be clear. everything in libnetwork 0.7.x is meant for 1.11 and not for 1.10 release branch.
After this commit, anything specific to 1.10 must go into the bump release branch (1.10.1).